tests/business-logic/User.test.ts

```diff
@@ -0,0 +1,84 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let hashedPassword: string;
+const email = "doe@email.com";
```

Comment:
can you use `...@example.com` addresses?

Reply:
done